NewsApp/NewsApp/Controller/HomeController.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  HomeController.swift
+//  NewsApp
+//
+//  Created by Josselin DUBOC on 26/03/2020.
+//  Copyright © 2020 Josselin DUBOC. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class HomeController: UITabBarController {
+    @IBOutlet private weak var button: UIButton!
```

Comment:
N'hésite pas a preciser a quoi sert le bouton par exemple : `btnProfile`

Reply:
Yes la convention sur iOS serait un truc du genre profile**Button** / profile**View** / profile**SonType**

Reply:
fix in "fix PR"